src/components/PopUp/PopUp.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react'
 
 interface PopUpProps {
   title: string
-  content: string
+  content: any
```

Comment:
Nit:
```suggestion
  children: any
```

Reply:
I leave it as content since it's the content of the PopUp for reusing purpose, for ex, and i think children is a reserved keyword so it'll cause linting error saying that u shouldn't pass "children" as props
```js
<PopUp
        title="Before you continue"
        content="FullyHacks only open to CSUF students 18 years old or older 🐘"
        action="I understand"
      />
```